hpx_tiles/casa_tiling.py

```diff
@@ -17,12 +17,98 @@
 import argparse
 import logging
 from casatasks import imhead, imregrid, exportfits
-
+from astropy import units as u
+from astropy_healpix import HEALPix
+from regions import Regions
+from astropy.io import fits
+from astropy import wcs
 
 logging.basicConfig(level=logging.INFO)
 
-
-def parse_args(argv):
+def tileID_to_region(tileID):
+    """@Erik Osinga
+    Input; tile ID
+    Returns: Regions object (i.e. region) denoting the tile with hpx number "tileID"
+    """
+    Nside=32
```

Comment:
Do you think it's worth making this a user-defined parameter provided with default as an argument?